flink-ai-flow/lib/notification_service/java/src/main/java/org/aiflow/notification/conf/Configuration.java

```diff
@@ -0,0 +1,18 @@
+package org.aiflow.notification.conf;
+
+import java.util.Properties;
+
+public class Configuration {
+
+    public static final String CLIENT_ENABLE_IDEMPOTENCE_CONFIG_KEY = "enable.idempotence";
```

Comment:
What is the default value of `enable.idempotence` config?

Reply:
False by default, you remind me to add some comments